### PR TITLE
Added section RE data protection/classification

### PR DESCRIPTION
**Pull request by bmuller**

### CODE REVIEW

**Review by artkay — APPROVED** [review]

**Review by tonypecc — APPROVED** [review]

**Review by nbgoss81 — APPROVED** [review]

File: security/data-protection.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ * Public announcements and blog posts
+
+## Yellow :full_moon:
+This is any data and information that should not be made publicly available that is created and used in the normal course of business. Unauthorized access or disclosure could cause minimal risk or harm and/or adversely impact Parallel Markets, its partners, employees, contractors, and customers.  This data can be stored on company devices, Google Workspace and other private cloud environments (so long as the data is not publicly accessible).
+
+Examples:
+
+ * General internal company communications
+ * Vendor contracts
+ * Internal policies or proceedures
```
Comment (noamsauerutley) [inline security/data-protection.md:33]:
```suggestion
 * Internal policies or procedures
```

**Review by noamsauerutley — APPROVED** [review]
👍

File: _sass/custom/custom.scss [hunk 2]
```diff
@@ -17,3 +17,11 @@ a.navigation-list-link.active {
 }
 
 @media (min-width: 31.25rem) { .site-title { font-size: 20px !important; } }
+
+blockquote {
+    background-color: #eee;
+    padding: 1px 20px 1px 20px;
+    border-radius: 5px;
+    border-left: thick double #188cf0; /* #32a1ce; */
```
Comment (RickDT) [inline _sass/custom/custom.scss:25]: What's the purpose of the commented-out color?

File: security/data-protection.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+## Green :green_book:
+This is any data that is publicly shareable and does not expose Parallel Markets or its customers to any harm or material impact.  This data can be stored on company systems but shared publicly.
+
+Examples:
+
+ * This handbook
+ * Our general product descriptions on [parallelmarkets.com](https://parallelmarkets.com)
+ * Public announcements and blog posts
+
+## Yellow :full_moon:
```
Comment (RickDT) [inline security/data-protection.md:26]: Full moon isn't yellow in all renderings:
https://emojipedia.org/full-moon/

Maybe 📒 or 🍯 or 🟡

**Review by RickDT — APPROVED** [review]
Please see comment about emoji for yellow.

File: security/data-protection.md (hunk 1 above)
Comment (bmuller) [reply security/data-protection.md:33]: @noamsauerutley good catch - this was copy/pasta from gitlab, sounds like their massive team didn't have your 🦅 👁️

File: _sass/custom/custom.scss (hunk 2 above)
Comment (bmuller) [reply _sass/custom/custom.scss]: No purpose, leftovers.  Removed in 8432068

**Review by nonrational — APPROVED** [review]

File: security/data-protection.md (hunk 3 above)
Comment (bmuller) [reply security/data-protection.md]: Good catch - fixed in 750a43a

Comment (nonrational) [reply security/data-protection.md]: 💛

**Pull request merged**